draft-ietf-teep-protocol.md

```diff
@@ -550,7 +550,7 @@ token
 : The value in the token field is used to match responses to requests.
 
 tc-list
-: The tc-list parameter enumerates the Trusted Components to be deleted,
+: The tc-list parameter enumerates the Trusted Components to be updated,
```

Comment:
I believe this wasn't a typo and the "fix" is incorrect.  The tc-list field should only contain the id's of trusted components to delete.  The manifest-list field identifies all the others to update, but that field needs the full manifest (which is required to add or update a component anyway).  The full manifest isn't needed to delete a component which is why the tc-list field is separate.

It would be ok to rename the tc-list for clarity, but we should not imply that the tc-list has to contain all ids in the manifest-list.

Reply:
Thanks, I will unify the name to unneeded-tc-list with holding SUIT_Component_Identifier also at the PR https://github.com/ietf-teep/teep-protocol/pull/94

Reply:
Closing this PR, the new PR is https://github.com/ietf-teep/teep-protocol/pull/97